docs/source/installation.rst

```diff
@@ -120,15 +121,17 @@ via PyPi
 --------
 .. code-block:: bash
 
-    virtualenv --system-site-packages $HOME/ve
+    python3 -m venv $HOME/ve
     source $HOME/ve/bin/activate
```

Comment:
I made the same comment on the EnTK doc fix you proposed: maybe we wait a bit and see how `venv` behaves before documenting it as the default / recommended deployment path?  Not saying it's not the right thing to do, but I'd rather make sure it is stable (it's new-ish code which recently had significant changes, and it is not feature-complete wrt. virtualenv).

Reply:
Fair, answered there. How do we want to track the TODOs? I would usually use a ticket but maybe it is time to give a try to the project feature of github?

Reply:
> How do we want to track the TODOs? I would usually use a ticket but maybe it is time to give a try to the project feature of github?

Sure, why not?